### PR TITLE
Fix: Markdown link checker

### PR DESCRIPTION
**Pull request by zee-hussain**
Correct the markdown link check so it does not fail on links to the GitHub doc site. The github doc site was returning 403s so 403 was added to `aliveStatusCodes` to prevent those from failing.

## Description

A custom configuration was added to the link checker tool that would treat 403s as a success.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] This PR's code follows as closely as possible [the coding style/guidelines of this project](???).
- [ ] I have added error reporting and followed [the error reporting guidelines](???).
- [ ] I have added event tracking and followed [the event tracking guidelines](???).
- [ ] I have updated any relevant documentation accordingly to reflect this PR's changes.
- [ ] I have added automated tests (unit/integration/e2e/other) to cover my changes.
- [ ] All new and existing tests pass locally (excluding flaky CI tests).

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "aliveStatusCodes": [200, 206, 403]
```
Comment (daniel-white) [inline config.json:2]: so i wonder if we should just do this for `github.com` urls... i'm guessing they are trying to prevent a feedback attack

Comment (daniel-white) [reply config.json:2]: (meaning, we just ignore github urls)

Comment (pamgoodrich) [reply config.json:2]: Agreed, if we can just do it for Github sites.

Comment (zee-hussain) [reply config.json]: Made the change @daniel-white @pamgoodrich

**Review by pamgoodrich — APPROVED** [review]

**Pull request merged**